data_safe_haven/resources/workspace/workspace.cloud_init.mustache.yaml

```diff
@@ -75,57 +107,17 @@ apt:
 
 # Install necessary apt packages
 packages:
-  # System requirements
+  - ansible
   - libnss-ldapd  # LDAP login
   - libpam-ldapd  # LDAP login
   - ldap-utils    # LDAP login
   - nfs-common    # Mounting shared volumes
-  - xfce4         # XFCE desktop
-  - xrdp          # remote desktop client
-  # Programming
-  - python-is-python3  # Python language
-  - python3            # Python language
-  - python3-dev        # Python language
-  - python3-pip        # Python language
-  - python3-venv       # Python language
-  - r-base             # R language
-  # Tools
-  - firefox       # web browser
-  - libpq-dev     # interact with PostgreSQL databases
-  - msodbcsql17   # interact with Microsoft SQL databases
-  - unixodbc-dev  # interact with Microsoft SQL databases
-  # Bash testing
-  - bats
 package_update: true
 package_upgrade: true
 
 runcmd:
-  # Configure LDAP user login
-  # -------------------------
-  - echo ">=== Configure LDAP user login... ===<"
-  # Configure nsswitch
-  - echo "Adding ldap to /etc/nsswitch.conf"
-  - sed -i -r "s|^(passwd\|group\|shadow)(:.*)$|\1\2 ldap|g" /etc/nsswitch.conf
-  # Configure PAM for LDAP
-  - echo "Configuring PAM for LDAP login"
-  - sed -i "s|pam_mkhomedir.so|pam_mkhomedir.so skel=/etc/skel umask=0022|g" /usr/share/pam-configs/mkhomedir
-  - sed -i "s| use_authtok||g" /usr/share/pam-configs/ldap
-  - pam-auth-update --enable mkhomedir ldap
-  # Allow SSH password authentication
-  - sed -i "s|^PasswordAuthentication.*|PasswordAuthentication yes|g" /etc/ssh/sshd_config
-  - sed -i "s|^UsePAM.*|UsePAM yes|g" /etc/ssh/sshd_config
   # Restart services
-  - systemctl restart nscd
   - systemctl restart nslcd
-  - systemctl restart sshd
-
-  # Enable xrdp and session manager at startup
-  # ------------------------------------------
-  - echo ">=== Enable xrdp... ===<"
-  - systemctl enable xrdp
-  - systemctl enable xrdp-sesman
-  - systemctl start xrdp
-  - systemctl start xrdp-sesman
 
   # Mount all external volumes
```

Comment:
Add a test for /desiredstate here too.

N.B. Is there a better place we can put this than as a new root-level directory? There must be an established location for config files that would be a better mount point.

Reply:
I would probably put all of the mounts in `/mnt` when is conventionally used for non-system disks at network drives. (`/media` or more recently `/run/media` for removable media).

Do you want to open an issue?

Reply:
Captured in #2027 and marked for rc3.